extensions/adobe/experience/customerJourneyManagement/message-experienceevent.schema.json

```diff
@@ -0,0 +1,106 @@
+  "description": "Adobe CJM ExperienceEvent mixin for ingesting the ExperienceEvent. Includes the core/standard ExperienceEvent as well as the required Adobe CJM ExperienceEvent mixins.",
+  "meta:extensible": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "meta:extends": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "definitions": {
+    "messageExperienceevent": {
+      "properties": {
+        "xdm:messageExecution": {
+          "title": "Message Execution",
+          "$ref": "https://ns.adobe.com/xdm/experience/customerJourneyManagement/messageexecution",
+          "description": "MessageExecution details for the Adobe CJM ExperienceEvent."
+        },
+        "xdm:messageProfile": {
+          "title": "Message Profile",
+          "$ref": "https://ns.adobe.com/xdm/experience/customerJourneyManagement/messageprofile",
+          "description": "MessageProfile details for the Adobe CJM ExperienceEvent."
+        },
+
+        "xdm:channelContext": {
+          "title": "Message Channel context",
+          "description": "Delivery Channel fields for the Messageprofle associated with the Adobe CJM Expereince Event.",
+          "oneOf": [
+            {
+              "properties": {
+                "xdm:emailChannelContext": {
+                  "title": "Email Channel Context",
+                  "$ref": "https://ns.adobe.com/xdm/experience/customerJourneyManagement/email-channel-context",
+                  "description": "Email channel fields for the Messageprofle associated with the Adobe CJM Expereince Event."
+                }
+              },
+              "required": ["xdm:emailChannelContext"]
+            },
+            {
+              "properties": {
+                "xdm:pushChannelContext": {
+                  "title": "Push Channel Context",
+                  "$ref": "https://ns.adobe.com/xdm/experience/customerJourneyManagement/push-channel-context",
+                  "description": "Push channel fields for the MessageProfile associated with the Adobe CJM Expereince Event."
+                }
+              },
+              "required": ["xdm:pushChannelContext"]
+            }
+          ]
+        },
+        "xdm:eventStatus": {
+          "title": "Event Status",
+          "description": "Provide the status for the Delivery, Feedback, Tracking Expereince Event for the MessageProfile.",
+          "oneOf": [
+            {
+              "properties": {
+                "xdm:messageDeliveryfeedback": {
+                  "title": "Message Profile",
+                  "$ref": "https://ns.adobe.com/xdm/experience/customerJourneyManagement/message-delivery-feedback",
+                  "description": "Provide status of a delivery attempt."
+                }
+              },
+              "required": ["xdm:messageDeliveryfeedback"]
+            },
+            {
+              "properties": {
+                "xdm:messageTrackingLink": {
```

Comment:
not sure about the name of property as we have push, open  action as well. Link seems we are only referring to tracking link. May be xdm:messageAction

Reply:
Agree.

Reply:
Right